src/auro-pane.js

```diff
@@ -35,18 +63,48 @@ class AuroPane extends LitElement {
     `;
   }
 
-  // function that renders the HTML and CSS into  the scope of the component
+  parseDateString() {
+    const dateFormatLength = 10;
+
+    /* eslint-disable no-magic-numbers, sort-vars */
+    if (this.date && this.date.length === dateFormatLength) {
+      // Using substring instead of date parsing due to browser inconsistencies
+      const year = this.date.substring(0, 4),
+        month = this.date.substring(5, 7),
+        date = this.date.substring(8),
+        parsedDate = new Date(year, month - 1, date);
+
+
+      return {
+        day: this.weekdays[parsedDate.getUTCDay()],
+        date: parsedDate.getUTCDate(),
+        month: this.months[parsedDate.getUTCMonth()]
+      };
+    }
+    /* eslint-enable no-magic-numbers, sort-vars */
+
+    return {};
+  }
+
   render() {
+    const classes = {
+      'auro-pane': true,
+      'auro-pane--selected': this.selected,
+      'auro-pane--disabled': this.disabled
+    },
+      parsedDate = this.parseDateString();
+
     return html`
-      <div class=${this.cssClass}>
-        <slot></slot>
-      </div>
+      <button class="${classMap(classes)}" ?disabled="${this.disabled}">
+        <div class="day-of-week">${parsedDate.day}</div>
```

Comment:
wrap this in a `if this.date` prior to render. Then add after line 100 a `<slot>` element for generic content. 

I am not sure it's worth the trouble to care if an empty `<slot>` appears in the DOM.

Reply:
I am having a hard time envisioning how slotted content should work. Does it replace date or would there be situations where we show date + slotted content? What about date + slotted content + price? How should slotted text be styled -- should it be the same color/size as day of week, date, or price? Are there limits on how much text can be slotted?

Reply:
Slotted content is a wild card. I am only introducing the idea as a way to be flexible with the content within the panel. For this MVP we don't need a slot as we don't have that use-case. But I would still consider only displaying the other DOM if `this.date` is not null.

Reply:
I believe we ended up deciding not to worry about slotted content for v1, so I'm resolving this.